libr/util/d/pokered

```diff
@@ -0,0 +1,61 @@
+"k anal/meta/0x7f=-"
```

Comment:
imho the sdb path shouldnt be in those files. it should be just 0x80=A without the `k anal/meta/`

Reply:
@radare In this case I can not keep the code with this sdb path in libr/core/cmd_print.c. I will change it too.

Reply:
@radare EDIT: this is done! This part works perfectly. I am going to push.